docs/modules/ROOT/pages/how-tos/aks_recover_from_backup.adoc

```diff
@@ -0,0 +1,141 @@
+
+=== Obtaining restic configuration from cluster
+
+[source,console]
+----
+export RESTIC_REPOSITORY=$(kubectl -n syn-cluster-backup get schedule objects -o jsonpath='azure:{.spec.backend.s3.bucket}:/')
+export RESTIC_PASSWORD=$(kubectl -n syn-cluster-backup get secret objects-backup-password -o jsonpath='{.data.password}' | base64 --decode)
+export AZURE_ACCOUNT_NAME=$(kubectl get secret admin-credentials-minio-k8up -n syn-minio -o jsonpath='{.data.accesskey}' | base64 --decode)
+export AZURE_ACCOUNT_KEY=$(kubectl get secret admin-credentials-minio-k8up -n syn-minio -o jsonpath='{.data.secretkey}' | base64 --decode)
+----
+
+=== Obtaining restic configuration from catalog and vault
+
+1. Obtain the repository URL of the clusters catalog and export it to `REPO_URL`
++
+Get the URL from https://control.vshn.net/syn/lieutenantclusters.
+Alternatively, get it from the Lieutenant API or the Kubernetes API Lieutenant is running on.
++
+.Use the Lieutenant API
+[source,console]
+----
+REPO_URL=$(curl -sH "${LIEUTENANT_AUTH}" "https://${LIEUTENANT_URL}/clusters/${CLUSTER_ID}" | jq -r .gitRepo.url)
+----
++
+.Use the Kubernetes API
+[source,console]
+----
+REPO_URL=$(kubectl -n ${LIEUTENANT_NS} get cluster -o jsonpath='{.spec.gitRepoURL}' ${CLUSTER_ID})
+----
+
+2. Download and extract the cluster catalog
++
+[source,console]
+----
+mkdir catalog
+git archive --remote ${REPO_URL} master | tar -xC catalog
+----
+
+3. Login to vault
++
+[source,console]
+----
+export VAULT_ADDR=https://vault-prod.syn.vshn.net
+vault login -method=ldap username=<your username>
+----
+
+4. Export restic configuration
++
+[source,console]
+----
+export RESTIC_REPOSITORY=azure:$(yq -r '. | select(.kind == "Schedule").spec.backend.s3.bucket' catalog/manifests/cluster-backup/05_schedule.yaml):/
+
+PASSWORD_KEY="$(yq -r '. | select(.kind == "Secret" and .metadata.name == "objects-backup-password").stringData.password' catalog/manifests/cluster-backup/05_schedule.yaml | cut -d: -f2)"
+export RESTIC_PASSWORD=$(vault kv get -format json "clusters/kv/${PASSWORD_KEY%/*}" | jq -r ".data.data.${PASSWORD_KEY##*/}")
+
+ID_KEY="$(yq -r '. | select(.kind == "Secret" and .metadata.name == "object-backup-s3-credentials").stringData.username' catalog/manifests/cluster-backup/05_schedule.yaml | cut -d: -f2)"
+export AZURE_ACCOUNT_NAME=$(vault kv get -format json "clusters/kv/${ID_KEY%/*}" | jq -r ".data.data.${ID_KEY##*/}")
+
+SECRET_KEY="$(yq -r '. | select(.kind == "Secret" and .metadata.name == "object-backup-s3-credentials").stringData.password' catalog/manifests/cluster-backup/05_schedule.yaml | cut -d: -f2)"
+export AZURE_ACCOUNT_KEY=$(vault kv get -format json "clusters/kv/${SECRET_KEY%/*}" | jq -r ".data.data.${SECRET_KEY##*/}")
+----
+
+== Identify and retrieve snapshot
+
+1. List the available snapshots.
+   Identify the one you do want to restore.
+   Take note of its `ID`.
++
+[source,console]
+----
+restic snapshots
+----
+
+2. Retrieve the backup archive
++
```

Comment:
You can achieve automatic numbering by simply having the dot without the number.

Also, the "console" style doesn't exist in asciidoctor (that's more Markdown style), might as well leave it empty. Use bash instead, as that will render a nicer output IMO.

there are multiple occurrences of my suggestion, but I'm not going to mark each now.
```suggestion
. List the available snapshots.
   Identify the one you do want to restore.
   Take note of its `ID`.
+
[source,bash]
----
restic snapshots
----

. Retrieve the backup archive
+
```

Reply:
As far as I understand the `console` style does exist for our Antora theme, and allows you to copy-paste commands when they're given with a shell prompt. I've observed that leading `#` or `$` are not selectable for blocks styled with `console`.

However, if you're giving commands without shell prompts, the style doesn't really matter.